src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -48,4 +50,12 @@ public List<IexLastTradedPrice> getLastTradedPrice(
     return iexService.getLastTradedPriceForSymbols(symbols);
   }
 
+  @GetMapping(value = "${mvc.iex.getHistoricalPricePath}", produces = {MediaType.APPLICATION_JSON_VALUE})
+  public List<IexHistoricalPrice> getHistoricalPrice(@RequestParam(value="symbols") final List<String> symbols, @RequestParam(value = "tp") final List<String> tp
```

Comment:
You should include a javadoc for this method, and consider using a more readable variable name than just 'tp'